src/zzz_od/api/routers/battle_assistant.py

```
@@ -0,0 +1,2509 @@
+    ## 使用示例
+    ```python
+    import requests
+    response = requests.get("http://localhost:8000/api/v1/battle-assistant/operation-debug/config")
+    config = response.json()
+    print(f"当前模板: {config['template_name']}")
+    print(f"重复模式: {'开启' if config['repeat_mode'] else '关闭'}")
+    ```
+    """
+    try:
+        # 从配置中获取当前模板名称和设置
+        # 如果配置不存在，返回默认配置
```

Ruff (0.12.2)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)